### ISSUE TITLE
How does this approach lead to functioning networking?

### ISSUE THREAD

**Issue by villevoutilainen** [NVIDIA/stdexec#26]
First things first; the Networking proposal is paying, at best, lip service to P0443. io_services are not P0443 executors, they are far more than that, they combine errors and successful results into one channel that's provided into the callbacks the user passes into the work-submission. 

We have lofty ideas how to senderandreceiverify Networking, but those ideas are, proposal-wise, pipe dreams, especially when we start talking about how to do chunked downloads with resumed downloads if an earlier download fails. I wonder how this approach would lead to being able to do that with Networking, and *when* this proposal would lead to being able to do that with Networking. 

I also wonder how compatible that would be with how ASIO does things, and why the standardization of Networking should even consider waiting for it, because frankly, none of this executors and senders and receivers work has materially changed how Networking really does its job, nor has any of this work really provided anything useful for Networking, other than some basic concepts that we hope to extend to be useful for Networking's needs. Asynchronous streams? Great, when will Networking be able to use them? Schedulers that can _actually_ schedule networking work, and do the various staged-chained operations mentioned in the other issue talking about executors? Great, when will Networking be able to use them?

In other words, why should Networking pretend to remain to be coupled with Executors, whatever they are, when it really currently isn't except on a very superficial second-class-citizen level, and can't be if it wants to ship in a reasonable time frame?

**Comment by brycelelbach** [comment]
On the 2021-06-30 task force telecon, I suggested we should perhaps incorporate some content like what Ville wrote here.

Specifically, I think we should highlight the fact that P0443 is not /really/ integrated with the Networking TS and that the Networking TS shouldn't be coupled with executors.

@mjgarland suggested that:
* The facility to actually execute something (e.g. `execute`) has not materially changed.
* The properties that were added in P0443 to support the Networking TS could be added to the Networking TS.

@mjgarland thinks that we should not argue that P0443 and the Networking TS were not integrated, but should instead enumerate the ways in which the P0443 and the Networking TS were integrated and illustrate that we have those covered. For example, compare the concept of `executor` and the concept of `scheduler`. The only difference is things like no-throw copy constructability. 

I agree that's a better approach.
